src/main/java/pl/zz/sharefood/food/mapper/impl/FoodMapper.java

```diff
@@ -0,0 +1,27 @@
+package pl.zz.sharefood.food.mapper.impl;
+
+import org.springframework.stereotype.Component;
+import pl.zz.sharefood.food.domain.Food;
+import pl.zz.sharefood.food.dto.FoodBaseDto;
+
+@Component
+public class FoodMapper implements pl.zz.sharefood.food.mapper.FoodMapper {
```

Comment:
Nazwij to FoodMapperImpl i nie będzie potrzebna ścieżka przy implemetnacji

Reply:
Oki